CHANGELOG.md

```diff
@@ -9,26 +9,26 @@ Stable Packages
 |--------------------------------------------|--------------------|----------------|
 | ContingentClaims.Core                      | 1.0.0              | 1.0.1          |
 | ContingentClaims.Lifecycle                 | 1.0.0              | 1.0.1          |
-| Daml.Finance.Account                       | 1.0.1              | 1.0.2          |
+| Daml.Finance.Account                       | 1.0.1              | 2.0.0          |
```

Comment:
I wonder if this should be considered a breaking change ... in the `Account` template there is no change in either fields or choices. Only a method is removed. I'd double check with @georg-da one more time during the team meeting on Monday :)